build.gradle

```diff
@@ -83,11 +77,45 @@ subprojects { project ->
         if (isGrailsPlugin) {
             apply plugin: "org.grails.grails-plugin"
         } else {
-            apply from: "${commonBuild}/common-project.gradle"
+            apply plugin: 'groovy'
+            apply plugin: 'java-library'
```

Comment:
The `groovy` plugin is already applied on line 71 and the `java-library` plugin on line 76.

Reply:
Updated